src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -604,18 +604,19 @@
                 rootContext.Policy,
                 target);
 
+            rootContext.Logger.AnalyzingTarget(context);
+            rootContext.Logger.LogMemoryUsage(context);
```

Comment:
>LogMemoryUsage [](http://example.com/codeflow?start=31&length=14)

What we should do here is consult data somewhere that tells us whether memory logging is enabled. If so, we should generate an informational notification, it's a tool execution notification.

Note that this all leads to a problem: how do users specify verbose output that is specific to a certain logging domain? i.e., they might not want to enable fully verbose logging but want to see memory messages only, or pdb trace messages only, etc. We should establish and follow a clear pattern for this.

Reply:
What we should do here is consult data somewhere that tells us whether memory logging is enabled.
We should establish and follow a clear pattern for this.
----- Yes, I found we have a `Trace`, I added it there.